draft-ietf-masque-connect-ip.md

```diff
@@ -1271,6 +1271,56 @@ ADDRESS_ASSIGN capsule is parsed. This allows modifications to address
 assignement to operate atomically. Similarly, extensions that modify routing
 SHOULD behave similarly with regards to the ROUTE_ADVERTISEMENT capsule.
 
+# Performance Considerations
+
+Bursty traffic can often lead to temporally-correlated packet losses; in turn,
+this can lead to suboptimal responses from congestion controllers in protocols
+running inside the tunnel. To avoid this, endpoints SHOULD strive to avoid
+increasing burstiness of IP traffic; they SHOULD NOT queue packets in order to
+increase batching.
```

Comment:
This is a somewhat confusing thing to recommend, because optimal NIC performance comes from a small amount of batching, i.e., the TSO / GSO batch size. But I agree we should not encourage large batching.

Reply:
Done